packages/gestalt/src/Badge.css

```diff
@@ -11,6 +11,31 @@
   white-space: nowrap;
 }
 
+.NewBadge {
+  composes: antialiased sansSerif from "./Typography.css";
+  composes: inlineBlock from "./Layout.css";
+  composes: rounding1 from "./Borders.css";
+  composes: fontWeightSemiBold from "./Typography.css";
+  color: var(--color-text-inverse);
+  font-size: 14px;
+  height: 20px;
+  padding: 2px 6px;
+  white-space: nowrap;
+}
+
+.info {
+  background-color: var(--g-blue);
+}
+
+.interactiveInfo {
+  background-color: var(--g-blue);
+}
+
```

Comment:
these can be deleted with the token addition

Reply:
DONE